dirty_cat/super_vectorizer.py

```diff
@@ -456,7 +453,7 @@ def fit_transform(self, X, y=None):
         if self.verbose:
             print(f'[SuperVectorizer] Assigned transformers: {self.transformers}')
 
-        check_input(X)
```

Comment:
Hum, is this intended? If so, we should actually delete the line rather than comment it out.

If this is intended, we should understand what consequences it has to remove the line.

Reply:
Yes, it's something I've [explained earlier in this thread](https://github.com/dirty-cat/dirty_cat/pull/297#issuecomment-1217702118) :)

Reply:
Ok, then just remove it completely